docs/install/vm-install/Debian.md

```
@@ -0,0 +1,270 @@
+  - Removing plugin /opt/jans/jetty/jans-config-api/custom/libs/gluu-flex-admin-ui-plugin.jar from Jans Config API Configuration
+  - Deleting /opt/jans/jetty/jans-config-api/custom/config/log4j2-adminui.xml
+  - Deleting /opt/jans/jetty/jans-config-api/custom/config/log4j2.xml
+  - Rewriting Jans CLI init file for plugins
+  - Deleting /var/www/html/admin
+Disabling script A51E-76DA
+Restarting Apache
+Restarting Jans Auth
+Restarting Janssen Config Api
+```
+
+### Uninstall Janssen Packages
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)